adapter/internal/oasparser/envoyconf/routes_with_clusters.go

```diff
@@ -215,6 +216,32 @@ func CreateRoutesWithClusters(mgwSwagger model.MgwSwagger, upstreamCerts map[str
 			routes = append(routes, routesP...)
 		}
 		return routes, clusters, endpoints, nil
+
+	}
+
+	if mgwSwagger.EndpointType == constants.AwsLambda {
+		for _, resource := range mgwSwagger.GetResources() {
+			amznResourceName := ""
+			for i, operation := range resource.GetOperations() {
+				value := model.ResolveAmznResourceName(operation.GetVendorExtensions())
+
+				if (i != 0) && (amznResourceName != value) {
+					logger.LoggerOasparser.Errorf("All Arns must be the same value.")
+				} else if value == "" {
+					logger.LoggerOasparser.Errorf("Arn cannot be empty.")
+				} else {
+					amznResourceName = value
+				}
+				resource.SetAmznResourceName(amznResourceName)
+			}
+			//TODO: interceptor ep
+			routesX, err := createRoutes(genRouteCreateParams(&mgwSwagger, resource, vHost, "", awslambdaClusterName, awslambdaClusterName, nil, nil, organizationID, false))
+			if err != nil {
+				//TODO: add errorcode
```

Comment:
Are we expecting to add error codes in the relevant sheet and place those in here ?

Reply:
Fixed. [https://github.com/wso2/product-microgateway/pull/3278/commits/3935734cf7ca9559802d74ff6c14589a12109d4b](https://github.com/wso2/product-microgateway/pull/3278/commits/3935734cf7ca9559802d74ff6c14589a12109d4b)